mockstore/mocktikv/cluster_manipulate.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 PingCAP, Inc.
```

Comment:
should change all licenses to 2019?

Reply:
No need to. The file is copied from tidb.

Reply:
Sadly, I have done the changes already...

Reply:
I think it does not matter to change it or not to :)